talk/owt/sdk/include/cpp/owt/p2p/p2pclient.h

```diff
@@ -222,6 +222,7 @@ class P2PClient final
   std::shared_ptr<P2PSignalingChannelInterface> signaling_channel_;
   std::unordered_map<std::string, std::shared_ptr<P2PPeerConnectionChannel>>
       pc_channels_;
+  std::vector<std::shared_ptr<P2PPeerConnectionChannel>> removed_pc_channels_;
```

Comment:
What is removed_pc_channels_ variable for?

Reply:
`removed_pc_channels` includes closed channels. Disposing `PeerConnectionChannel` immediately when ICE connection state changes to closed causes deadlock.